build.sbt

```diff
@@ -39,6 +40,10 @@ lazy val root = (project in file("."))
     buildInfoPackage := "prism",
     libraryDependencies ++= Seq(
       "com.google.code.findbugs" % "jsr305" % "2.0.0",
+      "software.amazon.awssdk" % "lambda" % awsVersionTwo,
+      "software.amazon.awssdk" % "auth" % awsVersionTwo,
+      "software.amazon.awssdk" % "sts" % awsVersionTwo,
+      "com.beust" % "jcommander" % "1.75", // deal with security vulnerability introduced by aws sdk
```

Comment:
Ideally, let's delete this dependency once the vulnerability in aws sdk v2 is fixed! I'm not sure how we can remind ourselves to do this? Perhaps by adding a TODO?

Reply:
yep TODO in the comments sounds good, maybe with a link to some post discussing it